diff --git a/Dimension3D/sources/windows/Scene.cpp b/Dimension3D/sources/windows/Scene.cpp
index 62b5499..3fa36d8 100644
--- a/Dimension3D/sources/windows/Scene.cpp
+++ b/Dimension3D/sources/windows/Scene.cpp
@@ -446,9 +446,9 @@ namespace dim
 		active = ImGui::IsWindowFocused();
 
 		ImGui::SetCursorPos(ImVec2(8.f, 27.f));
-		ImGui::Image(frame_buffer.get_texture().get_id(), ImVec2(static_cast<float>(temp.x), static_cast<float>(temp.y)), ImVec2(0.f, 1.f), ImVec2(1.f, 0.f));
+		ImGui::Image((void*) frame_buffer.get_texture().get_id(), ImVec2(static_cast<float>(temp.x), static_cast<float>(temp.y)), ImVec2(0.f, 1.f), ImVec2(1.f, 0.f));
 		ImGui::SetCursorPos(ImVec2(8.f, 27.f));
-		ImGui::Image(render_texture.getTexture().getNativeHandle(),
+		ImGui::Image((void*) render_texture.getTexture().getNativeHandle(),
 			ImVec2(static_cast<float>(render_texture.getTexture().getSize().x),
 			static_cast<float>(render_texture.getTexture().getSize().y)), ImVec2(0.f, 1.f), ImVec2(1.f, 0.f));
 
