diff --git a/Dimension3D/sources/opengl/Shader.cpp b/Shader.cpp
index f0e68fc..5a77763 100644
--- a/Dimension3D/sources/opengl/Shader.cpp
+++ b/Shader.cpp
@@ -80,7 +80,7 @@ namespace dim
 			"\n"
 			"void main()\n"
 			"{\n"
-			"	vec4 initial_color = (1 - u_textured) * u_material.color + u_textured * texture2D(u_texture, v_texcoord);\n"
+			"	vec4 initial_color = (1 - u_textured) * u_material.color + u_textured * texture(u_texture, v_texcoord);\n"
 			"\n"
 			"	if (u_material.illuminated == 1)\n"
 			"	{\n"
